projects/storefrontapp-e2e-cypress/cypress/integration/vendor/cds/profile-tag/default-events.core-e2e-spec.ts

```diff
@@ -529,6 +529,56 @@ describe('verifying X-Consent-Reference header addition to occ calls', () => {
   });
 });
 
+describe('Cart merging on login', () => {
+  const loginAlias = 'loginNotification';
+  beforeEach(() => {
+    cdsHelper.setUpMocks(strategyRequestAlias);
+    cy.intercept({
+      method: 'POST',
+      path: '**/users/current/loginnotification**',
+    }).as(loginAlias);
+    navigation.visitHomePage({
+      options: {
+        onBeforeLoad: profileTagHelper.interceptProfileTagJs,
+      },
+    });
+    profileTagHelper.waitForCMSComponents();
+  });
+
+  it('should send a CartSnapshot event when the cart got merged after a successful login', () => {
```

Comment:
Just a minor change: `should send a CartSnapshot event when a cart gets merged after a successful login` as it describes the test in present tense :P

Reply:
done